docs/source/release_notes.rst

```diff
@@ -3,15 +3,21 @@
 Release Notes
 -------------
 
-Future Release
-==============
+.. Future Release
```

Comment:
do we always have the `..` and space here?

Reply:
Yep! It's necessary as part of restructured text syntax for commenting: https://docutils.sourceforge.io/docs/ref/rst/restructuredtext.html#comments